src/Umbraco.Web.UI.Client/src/views/components/content/umb-content-node-info.html

```diff
@@ -2,21 +2,19 @@
 
     <div class="umb-package-details__main-content">
 
-        <umb-box ng-if="node.urls" data-element="node-info-urls">
+        <umb-box ng-if="currentUrls" data-element="node-info-urls">
             <umb-box-header title-key="general_links"></umb-box-header>
             <umb-box-content class="block-form">
                 <ul class="nav nav-stacked" style="margin-bottom: 0;">
-                    <li ng-repeat="url in node.urls">
+                    <li ng-repeat="url in currentUrls">
                         <span ng-if="url.isUrl">
                             <a href="{{url.text}}" target="_blank">
                                 <i class="icon icon-window-popin"></i>
                                 <span>{{url.text}}</span>
                             </a>
-                            <span ng-if="url.culture" style="font-size: 13px; color: #cccccc; margin-left: 20px;">{{url.culture}}</span>
```

Comment:
@madsrasmussen the problem with removing this is when we have invariant content underneath variant content, then we end up with a list of all urls based on the ancestor's culture. The node is invariant so all urls are listed. Before we'd show which culture each belonged to and now it just looks like this which gets even stranger when an ancestor culture is unpublished

![image](https://user-images.githubusercontent.com/1742685/50149518-8c9cf480-030f-11e9-9b21-07985c1b7112.png)